components/Header/Header.tsx

```diff
@@ -9,40 +9,62 @@ import HeaderCTA from "./HeaderCTA";
 import styles from "./Header.module.css";
 import Magnifier from "./assets/magnify.svg?react";
 import Link from "components/Link";
+import { EventBanner, EventProps } from "components/EventBanner";
+import eventData from "../../public/events.json";
 
 const Header = () => {
+  const [events] = useState<EventProps[]>(eventData);
+  const [screenWidth, setScreenWidth] = useState(900);
   const [isNavigationVisible, setIsNavigationVisible] =
     useState<boolean>(false);
   const toggleNavigaton = useCallback(() => {
     setIsNavigationVisible((value) => !value);
     blockBodyScroll(isNavigationVisible);
   }, [isNavigationVisible]);
 
+  const handleResize = () => {
+    setScreenWidth(window.innerWidth);
+  };
+
+  useEffect(() => {
+    window.addEventListener("resize", handleResize);
+
+    return () => {
+      window.removeEventListener("resize", handleResize);
+    };
+  }, [screenWidth]);
   return (
-    <header className={styles.wrapper}>
-      <a href="/" className={styles["logo-link"]}>
-        <Logo />
-      </a>
-      <div className={styles.searchLink}>
-        <NavSearch testid="mobile-search" aria-details="Search website" />
-      </div>
-      <HeadlessButton
-        onClick={toggleNavigaton}
-        className={styles.hamburger}
-        data-testid="hamburger"
-        aria-details="Toggle Main navigation"
-      >
-        <Icon name={isNavigationVisible ? "close" : "hamburger"} size="md" />
-      </HeadlessButton>
-      <div
-        className={cn(styles.content, {
-          [styles.visible]: isNavigationVisible,
-        })}
+    <>
+      <EventBanner events={events} />
+      <header
+        style={{ top: !!events.length && screenWidth < 900 ? "48px" : 0 }}
```

Comment:
Can this be handled with a `@media screen` query with some minimum/maximum width? I don't think we need resize event handlers here